tcms/utils/user.py

```diff
@@ -7,7 +7,7 @@ def delete_user(user):
         Delete user across DB schemas.
     """
     if hasattr(user, 'tenant_set'):
-        from django_tenants.utils import schema_context  # pylint: disable=import-error
+        from django_tenants.utils import schema_context  # pylint: disable=import-error, import-outside-toplevel
```

Comment:
import-error -> E0401
import-outside-toplevel -> C0415

this will make flake8 happy